cli/functionary/package.py

```diff
@@ -149,9 +151,11 @@ def list(ctx):
     for package in packages:
         name = package["name"]
         id = package["id"]
+        summary = package["summary"]
```

Comment:
For the CLI, let's just show the summary and not the description anymore in the list views.   Basically it'll be similar to the UI then and we don't have to worry about some super long full description causing the list output to get out of hand.

Reply:
Also, if there is no summary it ends up displaying "None" right now.  Maybe we can fix that.